test/org/sagebionetworks/bridge/models/ForwardCursorPagedResourceListTest.java

```diff
@@ -52,11 +79,25 @@ public void canSerialize() throws Exception {
         ForwardCursorPagedResourceList<AccountSummary> serPage = BridgeObjectMapper.get().readValue(node.toString(),
                 new TypeReference<ForwardCursorPagedResourceList<AccountSummary>>() {
                 });
-
-        assertEquals(page.getOffsetKey(), serPage.getOffsetKey());
-        assertEquals(page.getPageSize(), serPage.getPageSize());
-        assertEquals(page.getFilters().get("emailFilter"), serPage.getFilters().get("emailFilter"));
-        assertEquals(page.hasNext(), serPage.hasNext());
+        
+        assertEquals("nextOffsetKey", serPage.getNextPageOffsetKey());
+        assertEquals("nextOffsetKey", serPage.getOffsetKey());
+        assertEquals(startTime, serPage.getStartTime());
+        assertEquals(endTime, serPage.getEndTime());
+        assertEquals(startTime, serPage.getScheduledOnStart());
+        assertEquals(endTime, serPage.getScheduledOnEnd());
+        assertEquals((Integer)100, serPage.getPageSize());
+        assertEquals((Integer)2, serPage.getTotal());
+        assertTrue(serPage.hasNext());
```

Comment:
Moved to a separate test, deleted the offsetKeyCanBeNull test as it was entirely redundant with the new test.